lms/static/sass/discussion/elements/_actions.scss

```diff
@@ -141,18 +141,15 @@
 
     .action-label {
       @extend %t-copy-sub2;
-      display: inline-block;
+      display: none;
```

Comment:
I advocate for an equal experience.

If you don't want anyone to see it, why include it in the markup?